### PR TITLE
updated menu items and desktop logged in state

### PR DESCRIPTION
**Pull request by 8rianHo**
description:
- moved menu items to the right closer to the buttons
- changed username dropdown to a profile avatar (as it felt too wordy on desktop view)

image

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
+    domains: ["images.unsplash.com"],
```
Comment (lujasmine) [inline next.config.js:5]: Just curious - what does this do?

**Review by lujasmine — APPROVED** [review]
🍬

File: next.config.js (hunk 1 above)
Comment (8rianHo) [reply next.config.js:5]: yess, Next/Image seems to be very strict on what you can put in the src of an Image tag - if you want an external url you have to add it to this domain list
